client/src/pages/Availability/Records.tsx

```diff
@@ -0,0 +1,102 @@
+import { FormLabel, Typography } from '@mui/material';
+import Grid from '@mui/material/Grid';
+import FormGroup from '@mui/material/FormGroup';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Checkbox from '@mui/material/Checkbox';
+import Select from '@mui/material/Select';
+import MenuItem from '@mui/material/MenuItem';
+import { Field } from 'formik';
+
+interface Props {
+  name: string;
+  days: {
+    sunday: DayInformation;
+    monday: DayInformation;
+    tuesday: DayInformation;
+    wednesday: DayInformation;
+    thursday: DayInformation;
+    friday: DayInformation;
+    saturday: DayInformation;
+  };
+}
+
+interface DayInformation {
+  active: boolean;
+  startTime: string;
+  endTime: string;
+}
+export default function Records(props: any): JSX.Element {
+  return (
+    <Grid
+      container
+      sx={{
+        height: '4.5rem',
+        display: 'flex',
+        borderTop: '1px solid #dbdbdb',
+        opacity: 1,
+        margin: '1em 0 1em 1em',
+      }}
+    >
+      {props.available !== undefined ? (
+        <>
+          {Object.entries(props.available.days).map(([key, value]) => (
```

Comment:
Each child in a list needs to have a unique key prop. Keys help React identify which items have changed, are added, or are removed. They should be given to the elements inside the array to give the elements a stable identity.

Reply:
added the key for array element